<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_devices">
       <property name="text">
        <string>Devices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create_screenshot">
       <property name="text">
        <string>Screenshot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create_screenshot2">
       <property name="text">
        <string>Screenshot2</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QToolButton" name="btn_swipe_DU">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../lib/pictures/arrow.qrc">
           <normaloff>:/arrow/arrow-up.png</normaloff>:/arrow/arrow-up.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="btn_swipe_LR">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../lib/pictures/arrow.qrc">
           <normaloff>:/arrow/arrow-right.png</normaloff>:/arrow/arrow-right.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="btn_swipe_RL">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../lib/pictures/arrow.qrc">
           <normaloff>:/arrow/arrow-left.png</normaloff>:/arrow/arrow-left.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="btn_swipe_UD">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../lib/pictures/arrow.qrc">
           <normaloff>:/arrow/arrow-down.png</normaloff>:/arrow/arrow-down.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Tap</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="sb_pos_x"/>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sb_pos_y"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_tap">
          <property name="text">
           <string>Tap</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>83</width>
        <height>445</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_screenshot">
         <property name="text">
          <string>Screen</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_devices</tabstop>
  <tabstop>btn_create_screenshot</tabstop>
  <tabstop>btn_create_screenshot2</tabstop>
  <tabstop>btn_swipe_DU</tabstop>
  <tabstop>btn_swipe_RL</tabstop>
  <tabstop>btn_swipe_LR</tabstop>
  <tabstop>btn_swipe_UD</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../lib/pictures/arrow.qrc"/>
 </resources>
 <connections/>
</ui>
